### PR TITLE
retrieve salesforce details via ajax calls on page ready

### PR DESCRIPTION
**Pull request by QuarpT**
retrieve salesforce details via ajax calls on page ready
Afterwards we still need to move exact target details and hasCommented to ajax calls
This is part of the work to speed up page load on the admin frontend

picture 44

### CODE REVIEW

File: app/controllers/AccessUser.scala [hunk 1]
```diff
@@ -21,26 +21,30 @@ class AccessUser @Inject() (
...
-            formWithErrors,
-            request.flash.get("message"),
-            conf.baseProfileUrl + userId,
-            conf.baseAvatarUrl + userId,
-            Some(user)
-          )
+    adminApi.findIdentityUser(userId).map { user =>
+      val form = fillUserForm(user)
+      val error = request.flash.get("error")
+      val formWithErrors = if (error.isDefined) form.withGlobalError(error.getOrElse("Unknown")) else form
```
[the remark this answers is not in the document]
Comment (QuarpT) [reply app/controllers/AccessUser.scala]: 👍 
(that wasn't me btw 😆)

File: app/views/repermissionCheck.scala.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   message: Option[String],
   error: Option[String])(implicit request: RequestHeader, messages: MessagesProvider, config: Config)
 
-@main(pageTitle)(Some(searchBar(searchBarDescription))){
+@main(pageTitle)(Some(searchBar(searchBarDescription)))(){
```
Comment (rahilb) [inline app/views/repermissionCheck.scala.html]: what's the deal with the empty parens?

Comment (QuarpT) [reply app/views/repermissionCheck.scala.html]: `+@(title: String)(searchBar: Option[Html])(headHtml: Html = Html(""))(content: Html)(implicit request: RequestHeader, messages: Messages, config: Config)`
`headHtml: Html = Html("")`
I think twirl Html blocks have to be curried

File: public/javascript/edit-user.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+console.log("HELLO WORLD")
```
Comment (walaura) [inline public/javascript/edit-user.js]: can we take this out? 😹

Comment (QuarpT) [reply public/javascript/edit-user.js]: 😢

**Comment by Calum-Campbell** [comment]
Just having a look at this now, great work!

**Pull request merged**